src/components/Calculator.js

```diff
@@ -1,143 +1,133 @@
-                +
-              </button>
-            </td>
-          </tr>
-          <tr>
-            <td>
-              <button type="button" name="7" onClick={this.handleClick}>
-                7
-              </button>
-            </td>
-            <td>
-              <button type="button" name="8" onClick={this.handleClick}>
-                8
-              </button>
-            </td>
-            <td>
-              <button type="button" name="9" onClick={this.handleClick}>
-                9
-              </button>
-            </td>
-            <td>
-              <button type="button" className="operator" name="x" onClick={this.handleClick}>
-                x
-              </button>
-            </td>
-          </tr>
-          <tr>
-            <td>
-              <button type="button" name="4" onClick={this.handleClick}>
-                4
-              </button>
-            </td>
-            <td>
-              <button type="button" name="5" onClick={this.handleClick}>
-                5
-              </button>
-            </td>
-            <td>
-              <button type="button" name="6" onClick={this.handleClick}>
-                6
-              </button>
-            </td>
-            <td>
-              <button type="button" name="-" onClick={this.handleClick}>
-                -
-              </button>
-            </td>
-          </tr>
-          <tr>
-            <td>
-              <button type="button" name="1" onClick={this.handleClick}>
-                1
-              </button>
-            </td>
-            <td>
-              <button type="button" name="2" onClick={this.handleClick}>
-                2
-              </button>
-            </td>
-            <td>
-              <button type="button" name="3" onClick={this.handleClick}>
-                3
-              </button>
-            </td>
-            <td>
-              <button type="button" name="+" onClick={this.handleClick}>
-                +
-              </button>
-            </td>
-          </tr>
-          <tr>
-            <td colSpan={2}>
-              <button type="button" name="0" onClick={this.handleClick}>
-                0
-              </button>
-            </td>
-            <td>
-              <button type="button" name="." onClick={this.handleClick}>
-                .
-              </button>
-            </td>
-            <td>
-              <button type="button" name="=" onClick={this.handleClick}>
-                =
-              </button>
-            </td>
-          </tr>
-        </table>
-      </div>
-    );
-  }
-}
+  return (
+    <div className="calculator">
+      {next ? (
+        <div className="result">{next}</div>
+      ) : (
+        <div className="result">{total}</div>
+      )}
+      <table>
+        <tr>
+          <td>
+            <button type="button" name="AC" onClick={handleClick}>
+              AC
+            </button>
```

Comment:
- [ ] I suggest that after performing an operation and click `A/C` please display a `0` in the screen instead of an empty screeen